modules/trusted-firmware-m/tfm_boards/common/tfm_hal_platform.c

```diff
@@ -35,8 +35,8 @@
 #if defined(TFM_PARTITION_CRYPTO)
 static enum tfm_hal_status_t crypto_platform_init(void)
 {
-	int err = 0;
 #ifdef CONFIG_HAS_HW_NRF_CC3XX
+	int err = 0;
```

Comment:
When we add support for CONFIG_HW_UNIQUE_KEY_RANDOM on 54L we will get an error with this change.

Reply:
you are right, I will try to refactor a bit.